fk/secretsend.go

```diff
@@ -149,4 +187,19 @@ func encryptSecret(secret string, pgpKey *pgpkey.PgpKey) (string, error) {
 	return buffer.String(), nil
 }
 
+func makeFileHintsForFilename(filename string) *openpgp.FileHints {
```

Comment:
I was going to suggest returning a `FileHints` instead a `*FileHints`.

But I see that would make your caller-code slightly less compact, and given that `nil` is an acceptable argument into `openpgp.Encrypt(...)` I take it back !